website/preprints/tasks.py

```diff
@@ -76,22 +82,18 @@ def update_or_enqueue_on_preprint_updated(preprint_id, update_share=True, share_
 
 def update_preprint_share(preprint, old_subjects=None, share_type=None):
     if settings.SHARE_URL:
-        if not preprint.provider.access_token:
```

Comment:
Now in the unlikely event this is set to None or False, we use the default token instead.